website/components/remote-plugin-docs/utils/resolve-nav-data.js

```diff
@@ -159,6 +162,16 @@ async function resolvePluginEntryDocs(pluginConfigEntry) {
     }
   })
   //
+  navNodes.sort((a, b) => {
```

Comment:
Added this sort function - sorts by `title`, with the exception that items with the title `"Overview"` are always shown first.